src/custom/int.rs

```diff
@@ -147,6 +187,26 @@ impl<'c, H: HugrView> CodegenExtension<'c, H> for IntTypesCodegenExtension {
     ) -> Box<dyn EmitOp<'c, CustomOp, H> + 'a> {
         Box::new(NullEmitLlvm)
     }
+
+    fn supported_consts(&self) -> HashSet<TypeId> {
+        [TypeId::of::<ConstInt>()].into_iter().collect()
+    }
+
+    fn load_constant(
+        &self,
+        context: &mut EmitFuncContext<'c, H>,
+        konst: &dyn hugr::ops::constant::CustomConst,
+    ) -> Result<Option<BasicValueEnum<'c>>> {
+        let Some(k) = konst.downcast_ref::<ConstInt>() else {
+            return Ok(None);
+        };
+        let ty: IntType<'c> = context
+            .llvm_type(&k.get_type())?
+            .try_into()
+            .map_err(|_| anyhow!("Failed to get ConstInt as IntType"))?;
+        // TODO we don't know whether this is signed or unsigned
+        Ok(Some(ty.const_int(k.value_u(), false).into()))
```

Comment:
Is this an issue? I thought LLVM doesn't distinguish between signed/unsigned ints?